app/models/sms/mass_messenger.rb

```diff
@@ -14,6 +14,8 @@ def self.outgoing_service_name
 
   # checks for a valid adapter and raises an error it there is none
   def self.ensure_adapter
```

Comment:
Can you merge this class with Broadcaster please? I can't remember why it exists separately but it doesn't make sense anymore.